packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -104,7 +103,6 @@ library LibProving {
         TaikoData.TierProof memory _proof
     )
         internal
-        returns (uint8)
```

Comment:
I think this will also lead to another abi break change, shall we keep it? or all prover clients will break after upgrade.